woocommerce/woocommerce-ios
# Dashboard: Add support for Dynamic Type to "Number of orders to fulfill" section

Fix #897 

The fix is not completely fixing the issue. Notice how the bottom margin is ignored for regular font sizes:

Before and after, for regular font sizes:
![before-after-bottom](https://user-images.githubusercontent.com/2722505/56645142-ad0b0000-66af-11e9-9f42-da929cf45293.jpg)

On the bright side, this section supports large font sizes a bit better:
![before-after-large](https://user-images.githubusercontent.com/2722505/56645229-e17ebc00-66af-11e9-9642-dbe631dcb98f.jpg)

But, there is still something off with the bottom margin.

## Changes
- Removed some constraints from the container view controller.
- Removed some constraints from the `NewOrdersViewController`
- Make the `NewOrdersViewController` calculate its own `preferredContentSize`
- The parent view controller `DashboardViewController` recalculates the height of the `NewOrdersViewController` according to its preferredContentSize.

## Testing
- Checkout the branch, run the tests.
- Navigate to Dashboard
- Open the Accessibility Inspector (Xcode > Open Developer Tool > Accessibility Inspector)
- In the Accessibility Inspector, select the simulator and the Settings button:
<img width="279" alt="Screenshot 2019-04-24 at 10 26 01 AM" src="https://user-images.githubusercontent.com/2722505/56628194-72877000-667b-11e9-9752-3954fc34e269.png">
- Change the font size at will

@mindgraffiti @astralbodies Another case of the trees not letting me see the forest. There is something off, it has to be obvious, I can not see it. Please, send help.
